coverage/ecommerce-dashboard/app/features/dashboard/component/dashboard/dashboard.component.ts.html

```diff
@@ -0,0 +1,205 @@
+
+<!doctype html>
+<html lang="en">
```

Comment:
why components have .ts.html and index.html ?
it should have component.ts and component.html